File: Syncfusion_Sample/Windows/Tools.Windows/Samples/2.0/Tabs Package/VS 2008 Tab Splitter/CS/Main.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Syncfusion.Windows.Forms;
using Syncfusion.Windows.Forms.Tools;

namespace TabSplitterContainer_2005
{
    public partial class Main : Office2007Form
    {
        SplitWindowForm window1 = null, window2 = null;

        public Main()
        {
            InitializeComponent();

            window1 = new SplitWindowForm("Window1.xaml", "Code1.rtf");
            window1.MdiParent = this;
            window1.DesignWindow = Image.FromFile(Syncfusion.Windows.Forms.WinFormsUtils.FindFile("Resources", "Designer1.png"));
            window1.Show();

            window2 = new SplitWindowForm("Window2.xaml", "Code2.rtf");
            window2.MdiParent = this;
            window2.DesignWindow = Image.FromFile(Syncfusion.Windows.Forms.WinFormsUtils.FindFile("Resources", "Designer2.png"));
            window2.Show();
        }

        void tabbedMDIManager1_TabControlAdded(object sender, TabbedMDITabControlEventArgs args)
        {
            if (args.TabControl != null)
                args.TabControl.BorderVisible = true;
        }

        public SplitWindowForm ActiveWindow
        {
            get
            {
                if (this.ActiveMdiChild != null)
                    return (SplitWindowForm)this.ActiveMdiChild;
                else
                    return null;
            }
        }

        private void Main_Load(object sender, EventArgs e)
        {
            this.tabbedMDIManager1 = new Syncfusion.Windows.Forms.Tools.TabbedMDIManager(this.components);
            this.tabbedMDIManager1.AttachedTo = this;

            this.tabbedMDIManager1.ImageSize = new System.Drawing.Size(16, 16);
            this.tabbedMDIManager1.NeedUpdateHostedForm = false;
            this.tabbedMDIManager1.TabStyle = typeof(Syncfusion.Windows.Forms.Tools.TabRendererVS2008);
            this.tabbedMDIManager1.UseIconsInTabs = false;
            this.tabbedMDIManager1.TabControlAdded += new Syncfusion.Windows.Forms.Tools.TabbedMDITabControlEventHandler(this.tabbedMDIManager1_TabControlAdded);
            //Initial Settings
            this.ActivateMdiChild(window1);
        }
    }
}

File: Syncfusion_Sample/Windows/Tools.Windows/Samples/2.0/Tabs Package/VS 2008 Tab Splitter/VB/Main.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports Syncfusion.Windows.Forms
Imports Syncfusion.Windows.Forms.Tools

Namespace TabSplitterContainer_2005
    Partial Public Class Main : Inherits Office2007Form
		Private window1 As SplitWindowForm = Nothing, window2 As SplitWindowForm = Nothing

		Public Sub New()
			InitializeComponent()

			window1 = New SplitWindowForm("Window1.xaml", "Code1.rtf")
			window1.MdiParent = Me
			window1.DesignWindow = Image.FromFile(Syncfusion.Windows.Forms.WinFormsUtils.FindFile("Resources", "Designer1.png"))
			window1.Show()

			window2 = New SplitWindowForm("Window2.xaml", "Code2.rtf")
			window2.MdiParent = Me
			window2.DesignWindow = Image.FromFile(Syncfusion.Windows.Forms.WinFormsUtils.FindFile("Resources", "Designer2.png"))
            window2.Show()
        End Sub

        Public ReadOnly Property ActiveWindow() As SplitWindowForm
            Get
                If Not Me.ActiveMdiChild Is Nothing Then
                    Return CType(Me.ActiveMdiChild, SplitWindowForm)
                Else
                    Return Nothing
                End If
            End Get
        End Property

        Private Sub Main_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
            '
            'tabbedMDIManager1
            '
            Me.tabbedMDIManager1 = New Syncfusion.Windows.Forms.Tools.TabbedMDIManager(Me.components)
            Me.tabbedMDIManager1.AttachedTo = Me
            Me.tabbedMDIManager1.ImageSize = New System.Drawing.Size(16, 16)
            Me.tabbedMDIManager1.NeedUpdateHostedForm = False
            Me.tabbedMDIManager1.TabStyle = GetType(Syncfusion.Windows.Forms.Tools.TabRendererVS2008)
            Me.tabbedMDIManager1.UseIconsInTabs = False
            'Initial Settings
            Me.ActivateMdiChild(window1)
        End Sub

        Private Sub tabbedMDIManager1_TabControlAdded(ByVal sender As Object, ByVal args As TabbedMDITabControlEventArgs)
            If Not args.TabControl Is Nothing Then
                args.TabControl.BorderVisible = True
            End If
        End Sub
    End Class
End Namespace